torch/_export/converter.py

```diff
@@ -676,7 +679,15 @@ def convert_node(self, node: torch._C.Node):
         # matching converter for that.
         handler_func_name = ir_name_to_func_name(node_kind)
         handler_func = getattr(self, handler_func_name, self.convert_call_function_op)
-        handler_func(node)
+        
+        log_msg = f"Convert [{str(node)[:-1]}]\nConvert using [{handler_func.__name__}] "
```

Comment:
I think it would be better if we first log `f"Convert [{str(node)[:-1]}]"` here, and then after the try/except log the failure. That way during debugging w/ breakpoints, we can see the logging when stepping through